src/pages/settings/AboutPage/AboutPage.tsx

```diff
@@ -114,15 +121,15 @@ function AboutPage(props) {
 
     return (
         <IllustratedHeaderPageLayout
-            title={props.translate('initialSettingsPage.about')}
+            title={translate('initialSettingsPage.about')}
             onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
             illustration={LottieAnimations.Coin}
             backgroundColor={theme.PAGE_THEMES[SCREENS.SETTINGS.ABOUT].backgroundColor}
             overlayContent={overlayContent}
         >
             <View style={[styles.settingsPageBody, styles.ph5]}>
-                <Text style={[styles.textHeadline, styles.mb1]}>{props.translate('footer.aboutExpensify')}</Text>
-                <Text style={[styles.baseFontStyle, styles.mb4]}>{props.translate('initialSettingsPage.aboutPage.description')}</Text>
+                <Text style={[styles.textHeadline, styles.mb1]}>{translate('footer.aboutExpensify')}</Text>
+                <Text style={[styles.webViewStyles.baseFontStyle, styles.mb4]}>{translate('initialSettingsPage.aboutPage.description')}</Text>
```

Comment:
```suggestion
                <Text style={styles.mb4}>{translate('initialSettingsPage.aboutPage.description')}</Text>
```

If `styles.baseFontStyle` doesn't exist, is best to just remove it then